includes/virtual-machines-disks-encryption-at-host-restrictions.md

```diff
@@ -13,7 +13,7 @@ ms.custom:
 - Supported for 4k sector size Ultra Disks and Premium SSD v2.
 - Only supported on 512e sector size Ultra Disks and Premium SSD v2 if they were created after 5/13/2023.
     - For disks created before this date, [snapshot your disk](../articles/virtual-machines/disks-incremental-snapshots.md) and create a new disk using the snapshot.
-- Can't be enabled if Azure Disk Encryption (guest-VM encryption using bitlocker/DM-Crypt) is enabled on your virtual machines (VMs) or virtual machine scale sets.
+- Can't be enabled if migrating a virtual machine or a virtual machine scale set that is encrypted with Azure Disk Encryption, or has ever been encrypted with Azure Disk Encryption, to Encryption at Host. See [Azure Disk Encryption scenarios on Windows VMs](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/virtual-machines/windows/disk-encryption-windows.md#restrictions) for further restrictions and unsupported scenarios when using Azure Disk Encryption.
```

Comment:
@roygara Can we not reference another article? The reason I put that in there is that there's a whole load of other information which maybe relevant to some customers. Had this been referenced then it would have prevented one of my customers from moving away from ADE.